### ISSUE TITLE
TCP BTL triggers false detection

### ISSUE THREAD

**Issue by abouteiller** [ulfm-devel/ompi#21]
**[Original report](https://bitbucket.org/icldistcomp/ulfm2/issue/21) by Aurelien Bouteiller (Bitbucket: [abouteiller](https://bitbucket.org/abouteiller), GitHub: [abouteiller](https://github.com/abouteiller)).**

----------------------------------------

```
c00.cauchy:29481] [[9733,1],1] ompi_rbcast_bml_send_complete_cb: status -12
[c00.cauchy:29481] PML:OB1: the error handler was invoked by the tcp BTL for proc [[9733,1],0] with info Socket closed
[c00.cauchy:29481] [[9733,1],1] ompi: Process [[9733,1],0] failed (state = -57).
```

The modifications to render the TCP BTL resilient seem to be excessive and cause the BTL to trigger failure events for operations with normal processes in some instances.

**Comment by abouteiller** [comment]
**Original comment by Aurelien Bouteiller (Bitbucket: [abouteiller](https://bitbucket.org/abouteiller), GitHub: [abouteiller](https://github.com/abouteiller)).**

----------------------------------------

```
c00.cauchy:29481] [[9733,1],1] ompi_rbcast_bml_send_complete_cb: status -12
[c00.cauchy:29481] PML:OB1: the error handler was invoked by the tcp BTL for proc [[9733,1],0] with info Socket closed
[c00.cauchy:29481] [[9733,1],1] ompi: Process [[9733,1],0] failed (state = -57).
```

**Comment by abouteiller** [comment]
**Original comment by Aurelien Bouteiller (Bitbucket: [abouteiller](https://bitbucket.org/abouteiller), GitHub: [abouteiller](https://github.com/abouteiller)).**

----------------------------------------

Possibly a bug inherited from upstream. George is working on a fix for Open MPI.

**Comment by abouteiller** [comment]
**Original comment by Aurelien Bouteiller (Bitbucket: [abouteiller](https://bitbucket.org/abouteiller), GitHub: [abouteiller](https://github.com/abouteiller)).**

----------------------------------------

Problem solved in patches 0237a707 61c5954f
